cost/aws/object_storage_optimization/README.md

```diff
@@ -1,29 +1,24 @@
-## AWS Object Storage Optimization
- 
-### What it does
+# AWS Object Storage Optimization
```

Comment:
This file is missing the Prerequisite section and the AWS Permissions section should not include details about Cloud management.  See the other README files in the PR for good examples.

Reply:
Changes have been placed in all the 12 AWS policies.